include/ignition/gui/GuiEvents.hh

```diff
@@ -255,6 +255,24 @@ namespace ignition
         /// for this event.
         private: bool menuEnabled;
       };
+
+      class BlockOrbit : public QEvent
```

Comment:
What's the intended use case for this event? I don't see it being used.

Reply:
BlockOrbit is emitted when the transform mode is on. it used here https://github.com/ignitionrobotics/ign-gazebo/pull/854/files